### PR TITLE
Added Duplicate Caption Button

### PR DESCRIPTION
**Pull request by plavabe12**
Added a duplicate caption button

### CODE REVIEW

File: src/client/src/models/Scene.js [hunk 1]
```diff
@@ -229,6 +229,39 @@ const Scene = {
...
+    for (var i = 0; i < Scene.currentScene.captions.length; i++) {
+      if (caption_max_id < Scene.currentScene.captions[i].id) {
+        caption_max_id = Scene.currentScene.captions[i].id
+      }
+    }
+    captionToCopy.id = caption_max_id + 1
+    // Adds a new caption
+    Scene.currentScene.captions.push(captionToCopy)
+    Scene.reloadCaption = true
+    console.log("Hi")
```
Comment (JosephJindrich) [inline src/client/src/models/Scene.js]: Take this out please

**Review by JosephJindrich — CHANGES_REQUESTED** [review]
Currently when you duplicate, if you have unsaved changes they automatically save, and are applied to the duplicated captions.

File: src/client/src/models/Scene.js (hunk 1 above)
Comment (plavabe12) [reply src/client/src/models/Scene.js]: Done

**Comment by plavabe12** [comment]
Code Cleaned Up

**Review by JosephJindrich — CHANGES_REQUESTED** [review]
Hitting Duplicate caption now takes me to a 404 not found page. Also hitting the duplicate button on the caption card causes the scene to duplicate instead of the caption.

**Comment by JosephJindrich** [comment]
The comment about clicking the duplicate button taking me to a duplicated scene was wrong. Howver, it does take me to 404 not found.

**Comment by plavabe12** [comment]
@JosephJindrich bug appeared after taking out the saving function. Fixed.

**Review by JosephJindrich — APPROVED** [review]
Looks good

**Pull request merged**